.github/workflows/velox_be.yml

```diff
@@ -40,89 +40,91 @@ on:
 env:
   HTTP_PROXY_HOST: 10.240.252.16
   HTTP_PROXY_PORT: 911
-  DOCKER_REGISTRY_HOST: 10.0.2.4
+  DOCKER_REGISTRY_HOST: 10.0.0.25
   DOCKER_REGISTRY_PORT: 5000
-
+  PATH_TO_GLUTEN_TE: ./tools/gluten-te
```

Comment:
I think we should be able to use the Github environment variables

Reply:
Hongze once removed the use of github environment in a revert commit: https://github.com/oap-project/gluten/commit/385b252468dddfce43ccd1071961e9deb71fbaa5. I will check this with him.